### PR TITLE
Fix #1093: Improve `dcf_info` tool to optionally dump file contents.

### LINKED ISSUE #1093: Provide Tool to More Easily Examine the Contents of a Serialized .dcf File
I'm looking at two possible bugs that will require me to do a deeper analysis of .dcf packed info, so having a tool to dump the contents of those files to some kind of JSON like string would be helpful.

I've got a changeset that might do the job and will bring forward after this bug is filed.

That should help us discuss possible bugs in .dcf file contents in an objective manner.

### PR DESCRIPTION
**Pull request by johnoneil**
See associated Issue for description.

Tim told me he's also working in this area, so I can close this as unnecessary if an improved workflow is incoming.

### CODE REVIEW

File: crates/figma_import/src/bin/dcf_info.rs [hunk 1]
```diff
@@ -12,11 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Utility program to parse a .dcf file and print out the header and version info for the file.
+// Utility program to parse a .dcf file and print its contents.
+// By default prints the header and file info.
+// Provide the optional `--dump` switch to dump the entire file contents.
```
Comment (johnoneil) [inline crates/figma_import/src/bin/dcf_info.rs:17]: Fix comment. Should read `--node` or `-n`.

Comment (johnoneil) [reply crates/figma_import/src/bin/dcf_info.rs]: fixed.

File: crates/figma_import/src/bin/dcf_info.rs [hunk 2]
```diff
@@ -12,11 +12,17 @@
...
 
-// Utility program to parse a .dcf file and print out the header and version info for the file.
+// Utility program to parse a .dcf file and print its contents.
+// By default prints the header and file info.
+// Provide the optional `--node` or `-n` switch to dump figma document using
+// that node as root.
+// Example:
+// `cargo run --bin dcf_info --features="dcf_info" -- tests/layout-unit-tests.dcf`
+// or
+// `cargo run --bin dcf_info --features="dcf_info" -- tests/layout-unit-tests.dcf -n HorizontalFill``
```
Comment (rylin8) [inline crates/figma_import/src/bin/dcf_info.rs:22]: Think you've got an extra ` mark at the end

**Comment by rylin8** [comment]
FYI @yiqunw700

File: crates/figma_import/src/bin/dcf_info.rs (hunk 2 above)
Comment (johnoneil) [reply crates/figma_import/src/bin/dcf_info.rs]: fixed.

**Review by rylin8 — APPROVED** [review]

**Comment by johnoneil** [comment]
Thanks @rylin8 for moving on this quickly.

I've been running this on some additional docs and am really pleased with the results. Allows me to find info I'm looking for very quickly.

Some examples:

```
cargo run --bin dcf_info --features="dcf_info" -- ./integration-tests/validation/build/intermediates/assets/debug/mergeDebugAssets/figma/LayoutTestsDoc_Gv63fYTzpeH2ZtxP4go31E.dcf -n "#stage" | tee layout.stage.output.txt

cargo run --bin dcf_info --features="dcf_info" -- ./integration-tests/validation/src/main/assets/figma/FancyFillTestDoc_xQ9cunHt8VUm6xqJJ2Pjb2.dcf -n "#stage" | tee fancy.fills.stage.output.txt
```

**Pull request merged**